packages/component-submission/client/pages/SubmissionWizard.js

```diff
@@ -35,7 +35,7 @@ import {
   getErrorStepsFromErrors,
   convertArrayToReadableList,
 } from '../utils'
-import { STEP_NAMES } from '../utils/constants'
+import { STEP_NAMES, STEP_TITLES } from '../utils/constants'
```

Comment:
Did you consider changing to something like `STEP_INFO` so we can add any future information to this structure. Seems like adding an additional structures each time might become messy?

Reply:
I did think about having just a STEP_CONF value that held everything. This would touch lots of other values though so maybe better too refactor in another issue?

Reply:
yup -raise it as an improvement :)

Reply:
#2180 raised as an improvement